### PR TITLE
Remove UI

### PR DESCRIPTION
**Pull request by aulneau**
### What this does
Removes UI that is now located in `@blockstack/connect`. Additionally, this improves and fixes some bugs that were introduced from updating the `@blockstack/ui` library.

### How to run
Clone this branch, `yarn && yarn dev`

Clone [blockstack/connect/feature/add-ui](https://github.com/blockstack/connect/tree/feature/add-ui), `yarn && cd test-app && yarn && yarn start`

### CODE REVIEW

File: src/ts/components/sign-up/header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import { Logo } from '@components/logo';
+import { Image } from '@components/image';
+
+interface AppIconProps {
+  src?: string;
+  name?: string;
+}
+
+const AppIcon: React.FC<AppIconProps> = ({ src, name, ...rest }) => (
```
Comment (hstove) [inline src/ts/components/sign-up/header/index.tsx]: Should this be removed in favor of the dedicated `AppIcon` component?

File: src/ts/components/sign-up/onboarding/screens/index.tsx [hunk 2]
```diff
@@ -1,450 +1,8 @@
...
-          </Flex>
-          <Link>Help</Link>
-        </>
-      }
-    />
-  );
-};
-
-export { Intro, HowItWorks, Create, SecretKey, Connect, SaveKey, Final, SignIn };
+import { SignIn } from '@components/sign-up/onboarding/screens/sign-in';
```
Comment (hstove) [inline src/ts/components/sign-up/onboarding/screens/index.tsx]: Nit: could use the `export { x } from ...` syntax and remove the export at the end

**Review by hstove — APPROVED** [review]
Minor questions / nits. This is good!

**Review by kyranjamie — APPROVED** [review]

**Comment by kyranjamie** [comment]
Given the other PR with folder structures changes has been merged into this one, I'm gonna merge now.

**Pull request merged**